src/storage/v2/id_types.hpp

```diff
@@ -31,6 +32,11 @@ namespace memgraph::storage {
     static name FromInt(int64_t id) { return name{utils::MemcpyCast<uint64_t>(id)}; }                         \
     uint64_t AsUint() const { return id_; }                                                                   \
     int64_t AsInt() const { return utils::MemcpyCast<int64_t>(id_); }                                         \
+    static name FromString(std::string_view id) {                                                             \
+      uint64_t value;                                                                                         \
+      std::from_chars(id.data(), id.data() + id.size(), value);                                               \
```

Comment:
What do you think about checking return values of `from_chars`?
I wrote checks in https://github.com/memgraph/memgraph/blob/master/src/utils/rocksdb_serialization.hpp#L214
I think there we also need to change to `FromString()` function call

Reply:
The reason I went without check is the performance. Check will obviously slow down things and I think that if something bad happens, std::from_chars will throw exception automatically.